### PR TITLE
Placehold Lat/Lon/Time for P16 Profiler mooring deployments.

### PR DESCRIPTION
**Pull request by snwhite**
Placeholder information needed for Lat, Lon and Start time for Profiler Moorings to be deployed on Leg 2 in order to push to OceanObservatories/asset-management for updates from Leg 1.

### CODE REVIEW

File: deployment/CP02PMCI_Deploy.csv [hunk 1]
```diff
@@ -125,12 +125,12 @@ AR44,,,,CP02PMCI-WFP01-02-DOFSTK000,14,1,2020-06-12T13:40:00,,CGMCP-02PMCI-00014
...
+AR52,,,,CP02PMCI-RII01-02-ADCPTG010,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,,CGINS-ADCPTG-19151,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-SBS01-00-STCENG000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,,CGCON-BSTC01-00017,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-SBS01-01-MOPAK0000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,,CGINS-MOPAK0-58478,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-WFP01-00-WFPENG000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-WFP01-01-VEL3DK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-WFP01-02-DOFSTK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-WFP01-03-CTDPFK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-WFP01-04-FLORTK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,40.2,-70.8,,,,,,
+AR52,,,,CP02PMCI-WFP01-05-PARADK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCI-00015,CGVEH-CWFPCP-99101,CGINS-PARADK-20541,40.2,-70.8,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMCI_Deploy.csv:136]: placeholders look good

File: deployment/CP02PMCO_Deploy.csv [hunk 2]
```diff
@@ -125,12 +125,12 @@ AR44,,,,CP02PMCO-WFP01-02-DOFSTK000,14,1,2020-06-10T13:53:00,,CGMCP-02PMCO-00014
...
+AR52,,,,CP02PMCO-RII01-02-ADCPTG010,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,,CGINS-ADCPTG-20495,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-SBS01-00-STCENG000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,,CGCON-BSTC01-00016,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-SBS01-01-MOPAK0000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,,CGINS-MOPAK0-43263,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-WFP01-00-WFPENG000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGCON-CWFPCP-99102,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-WFP01-01-VEL3DK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-VEL3DK-00013,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-WFP01-02-DOFSTK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-DOFSTK-02499,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-WFP01-03-CTDPFK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-CTDPFK-00113,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-WFP01-04-FLORTK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-FLORTK-01031,40.0,-70.8,,,,,,
+AR52,,,,CP02PMCO-WFP01-05-PARADK000,15,1,2021-04-08T00:00:00,,CGMCP-02PMCO-00015,CGVEH-CWFPCP-99102,CGINS-PARADK-20447,40.0,-70.8,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMCO_Deploy.csv:136]: placeholders look good

File: deployment/CP02PMUI_Deploy.csv [hunk 3]
```diff
@@ -134,12 +134,12 @@ AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00
...
+AR52,,,,CP02PMUI-RII01-02-ADCPTG010,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,,CGINS-ADCPTG-24543,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-SBS01-00-STCENG000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,,CGCON-BSTC01-00042,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-SBS01-01-MOPAK0000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,,CGINS-MOPAK0-43252,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-WFP01-00-WFPENG000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-WFP01-01-VEL3DK000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-WFP01-02-DOFSTK000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-WFP01-03-CTDPFK000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-WFP01-04-FLORTK000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,40.3,-70.7,,,,,,
+AR52,,,,CP02PMUI-WFP01-05-PARADK000,16,1,2021-04-08T00:00:00,,CGMCP-02PMUI-00016,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,40.3,-70.7,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMUI_Deploy.csv:145]: placeholders look good

File: deployment/CP02PMUO_Deploy.csv [hunk 4]
```diff
@@ -143,12 +143,12 @@ AR48,,,,CP02PMUO-WFP01-02-DOFSTK000,16,1,2020-11-05T15:23:00,,CGMCP-02PMUO-00016
...
+AR52,,,,CP02PMUO-RII01-02-ADCPSL010,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,,CGINS-ADCPSL-18353,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-SBS01-00-STCENG000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,,CGCON-BSTC01-00043,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-SBS01-01-MOPAK0000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,,CGINS-MOPAK0-70027,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-WFP01-00-WFPENG000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGCON-CWFPCP-10304,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-WFP01-01-VEL3DK000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-VEL3DK-00033,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-WFP01-02-DOFSTK000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-DOFSTK-02726,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-WFP01-03-CTDPFK000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-CTDPFK-00127,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-WFP01-04-FLORTK000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-FLORTK-01100,39.9,-70.7,,,,,,
+AR52,,,,CP02PMUO-WFP01-05-PARADK000,17,1,2021-04-08T00:00:00,,CGMCP-02PMUO-00017,CGVEH-CWFPCP-10304,CGINS-PARADK-20582,39.9,-70.7,,,,,,
```
Comment (cdobs) [inline deployment/CP02PMUO_Deploy.csv:154]: placeholders look good

File: deployment/CP04OSPM_Deploy.csv [hunk 5]
```diff
@@ -109,12 +109,12 @@ AR44,,,,CP04OSPM-WFP01-02-DOFSTK000,13,1,2020-06-08T13:50:00,,CGMCP-04OSPM-00013
...
+AR52,,,,CP04OSPM-RII01-02-ADCPSL010,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,,CGINS-ADCPSL-21393,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-SBS01-00-STCENG000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,,CGCON-BSTC01-00007,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-SBS11-02-MOPAK0000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,,CGINS-MOPAK0-11347,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-WFP01-00-WFPENG000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGCON-CWFPCP-99105,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-WFP01-01-VEL3DK000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-VEL3DK-00212,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-WFP01-02-DOFSTK000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-DOFSTK-03370,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-WFP01-03-CTDPFK000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-CTDPFK-00169,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-WFP01-04-FLORTK000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-FLORTK-01466,39.9,-70.8,,,,,,
+AR52,,,,CP04OSPM-WFP01-05-PARADK000,14,1,2021-04-08T00:00:00,,CGMCP-04OSPM-00014,CGVEH-CWFPCP-99105,CGINS-PARADK-20542,39.9,-70.8,,,,,,
```
Comment (cdobs) [inline deployment/CP04OSPM_Deploy.csv:120]: placeholders look good

**Review by cdobs — APPROVED** [review]
placeholders look good

File: deployment/CP02PMCI_Deploy.csv (hunk 1 above)
Comment (IDCalTech) [reply deployment/CP02PMCI_Deploy.csv:136]: looks good

File: deployment/CP02PMCO_Deploy.csv (hunk 2 above)
Comment (IDCalTech) [reply deployment/CP02PMCO_Deploy.csv:136]: looks good

File: deployment/CP02PMUI_Deploy.csv (hunk 3 above)
Comment (IDCalTech) [reply deployment/CP02PMUI_Deploy.csv:145]: looks good

File: deployment/CP02PMUO_Deploy.csv (hunk 4 above)
Comment (IDCalTech) [reply deployment/CP02PMUO_Deploy.csv:154]: looks good

File: deployment/CP04OSPM_Deploy.csv (hunk 5 above)
Comment (IDCalTech) [reply deployment/CP04OSPM_Deploy.csv:120]: looks good

**Review by IDCalTech — APPROVED** [review]
all placeholders have been reviewed, everything looks good

**Pull request merged**